### PR TITLE
Default variables for runtime environments

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: fixtures/base/10_runtimes.neon [hunk 1]
```diff
@@ -15,6 +20,11 @@ App\Model\Entity\RuntimeEnvironment:
             - "[pas]"
             - "GNU/Linux"
             - "Pascal compiled with FreePascal"
+            - '''
+              - name: "source_files"
+                type: "file[]"
```
Comment (Teyras) [inline fixtures/base/10_runtimes.neon]: This should accept a single file (as long as we don't want to implement building complex pascal apps)

File: tests/Presenters/RuntimeEnvironmentsPresenter.phpt [hunk 2]
```diff
@@ -26,7 +26,7 @@ class TestRuntimeEnvironmentsPresenter extends Tester\TestCase
   private $presenterPath = "V1:RuntimeEnvironments";
 
   /** @var RuntimeEnvironments */
-  protected $hardwareGroups;
+  protected $runtimeEnvironments;
```
Comment (Teyras) [inline tests/Presenters/RuntimeEnvironmentsPresenter.phpt]: Context? Is that just a fix of bad naming?

Comment (Neloop) [reply tests/Presenters/RuntimeEnvironmentsPresenter.phpt]: Just a fix which I ran into, while checking this presenter, somehow it worked without this variable.

File: fixtures/base/10_runtimes.neon (hunk 1 above)
Comment (Neloop) [reply fixtures/base/10_runtimes.neon]: Will be repaired in near future, together with appropriate box.

Comment (Teyras) [reply fixtures/base/10_runtimes.neon]: OK

**Pull request merged**